packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -0,0 +1,150 @@
+            ui.toggleKycInformationSubmittedModal();
+        } else if (status === 'error') {
+            setError?.(error);
+        }
+    }, [error, notifications, setError, status, ui, updateAccountStatus]);
+
+    if (isAccountSettingsLoading) {
+        return <Loading is_fullscreen={false} />;
+    }
+
+    return (
+        <Formik
+            validateOnMount
+            validateOnBlur
+            validateOnChange
+            initialValues={initialValues}
+            onSubmit={onSubmit}
+            validate={validate}
+        >
+            {({ isValid, dirty, setFieldValue }) => (
+                <Form className='additional-kyc-info-modal__form-layout'>
+                    {isLoading ? (
+                        <Loading is_fullscreen={false} />
+                    ) : (
+                        <section className='additional-kyc-info-modal__form-layout--fields'>
+                            <FormTitle />
+                            <fieldset className='additional-kyc-info-modal__form-field'>
+                                <FormSelectField {...fields.place_of_birth} />
+                            </fieldset>
+                            <fieldset
+                                className={classNames(
+                                    'additional-kyc-info-modal__form-field',
+                                    'additional-kyc-info-modal__form-field--info'
+                                )}
+                            >
+                                <FormSelectField {...fields.tax_residence} />
+                                <FormFieldInfo
+                                    message={
+                                        <Localize i18n_default_text='The country in which you meet the criteria for paying taxes. Usually the country in which you physically reside.' />
+                                    }
+                                />
+                            </fieldset>
+                            <fieldset className='additional-kyc-info-modal__form-field--info'>
+                                <FormInputField {...fields.tax_identification_number} />
+                                <FormFieldInfo
+                                    message={
+                                        <Localize
+                                            i18n_default_text="Don't know your tax identification number? <1 />Click <0>here</0> to learn more."
+                                            components={[
+                                                <a
+                                                    key={0}
+                                                    className='link'
+                                                    target='_blank'
+                                                    rel='noopener noreferrer'
+                                                    href='https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/'
```

Comment:
Don't hard code a url in the code, add it to a constant file.

Reply:
Updated. 
Added an external url constants file.